From 63e6253895e7173598da6ec29d101291e83cc8a3 Mon Sep 17 00:00:00 2001
From: Sultan Qasim Khan <sultanqasim@lineageos.org>
Date: Fri, 3 Aug 2018 00:27:53 -0400
Subject: [PATCH 220/249] wiki: update maintainer for osprey and surnia

I have always been the surnia maintainer, and I've taken over osprey since
Alberto stepped down.

Change-Id: I75816e7d12c3d2b161d1a11454367dcb43ab6232
---
 _data/devices/osprey.yml | 2 +-
 _data/devices/surnia.yml | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/_data/devices/osprey.yml b/_data/devices/osprey.yml
index edf86d6..a2f4965 100644
--- a/_data/devices/osprey.yml
+++ b/_data/devices/osprey.yml
@@ -17,7 +17,7 @@ gpu: Adreno 306
 image: osprey.png
 install_method: fastboot_motorola
 kernel: android_kernel_motorola_msm8916
-maintainers: [alberto97]
+maintainers: [sultanqasim]
 models: [XT1540, XT1541, XT1542, XT1543, XT1544, XT1548, XT1550]
 name: Moto G 2015
 network:
diff --git a/_data/devices/surnia.yml b/_data/devices/surnia.yml
index 252f0ed..93e2d27 100644
--- a/_data/devices/surnia.yml
+++ b/_data/devices/surnia.yml
@@ -18,7 +18,7 @@ height:
 image: otus.png
 install_method: fastboot_motorola
 kernel: android_kernel_motorola_msm8916
-maintainers: [alberto97]
+maintainers: [sultanqasim]
 models: [XT1514, XT1521, XT1523, XT1524, XT1526, XT1527]
 name: Moto E 2015 LTE
 network:
-- 
2.17.1

